### ISSUE TITLE
coc-pyright uses system python instead of virtual env python as interpreter

### ISSUE THREAD

**Issue by CharlieCheckpt** [fannheyward/coc-pyright#690]
Hello, 
Thank you for this great package.
I have created a virtual environment, but coc-pyright does not detect it, and use system python instead.
Yet, I source the env before launching (neo)vim. 

**What's the output of `:CocCommand pyright.version`**
coc-pyright 1.1.194 with Pyright 1.1.202

**What's the output of `:CocCommand workspace.showOutput Pyright`**
```
Workspace: /HOMES/chs/workspace/my_proj
Using python from /fusionio/opt/anaconda3/bin/python

[Info  - 9:39:51 PM] Pyright language server 1.1.202 starting
[Info  - 9:39:51 PM] Server root directory: /fusionio/home/chs/.config/coc/extensions/node_modules/coc-pyright/node_modules/pyright/dist/
[Info  - 9:39:51 PM] No configuration file found.
[Info  - 9:39:51 PM] No pyproject.toml file found.
[Info  - 9:39:51 PM] Setting pythonPath for service "my_proj": "/fusionio/opt/anaconda3/bin/python"
[Warn  - 9:39:51 PM] stubPath /HOMES/chs/workspace/my_proj/typings is not a valid directory.
[Info  - 9:39:52 PM] Assuming Python version 3.6
[Info  - 9:39:52 PM] Assuming Python platform Linux
```

Any idea what could be causing this ?

**Comment by last-partizan** [comment]
Have you activated your virtuial environment before starting nvim?

**Comment by fannheyward** [comment]
> Using python from /fusionio/opt/anaconda3/bin/python

coc-pyright uses this Python, not system one.

**Comment by CharlieCheckpt** [comment]
@last-partizan Thank you for your answer, yes I did. 

@fannheyward Thanks for your answer too. Okay, I meant this is the interpreter used when I launch python without any environment activated on my server. 

I managed to solve the issue after looking at this issue from [pyright](https://github.com/microsoft/pyright/issues/30#issue-426840109) repository.
For some reasons, pyright could not detect my virtualenv that was located in the root of my project. I created a new virtual environment somewhere else and now it works. Now, coc-pyright says it is using the python interpreter from my (new) environment. 

Hence, In case this helps other people with same problems, try to move your virtual environment somewhere else if it is in the project root.
